apiserver/facades/client/charms/interfaces/state.go

```diff
@@ -8,19 +8,10 @@ import (
 
 	"github.com/juju/juju/core/constraints"
 	"github.com/juju/juju/core/instance"
-	"github.com/juju/juju/environs/config"
 	"github.com/juju/juju/internal/charm/services"
 	"github.com/juju/juju/state"
 )
 
-type BackendModel interface {
-	Config() (*config.Config, error)
-	ModelTag() names.ModelTag
-	CloudRegion() string
-	ControllerUUID() string
-	Type() state.ModelType
-}
-
 type BackendState interface {
```

Comment:
This struct ad a `ModelUUID` call on it. Is it worth blowing this away in the area?

Reply:
done